client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -1,179 +0,0 @@
-import debugFactory from 'debug';
-const debug = debugFactory( 'calypso:wpcom-undocumented:site' );
-
-/**
- * Resources array
- * A list of endpoints with the same structure
- * [  wpcom-undocumented.functionName, siteAPiSubPath, apiVersion ]
- */
-const resources = [
-	[ 'statsEvents', 'posts/' ],
```

Comment:
The `statsEvents` one was never used?

Reply:
Good question - yup, I verified it wasn't in use anymore and removed it.